en/docs/chapter_hashing/hash_map.md

```diff
@@ -1,30 +1,30 @@
 # Hash table
 
-A <u>hash table</u> achieves efficient element querying by establishing a mapping between keys and values. Specifically, when we input a `key` into the hash table, we can retrieve the corresponding `value` in $O(1)$ time.
+A <u>hash table</u>, also known as a <u>hash map</u>, is a data structure that establishes a mapping between keys and values, enabling efficient element retrieval. Specifically, when we input a `key` into the hash table, we can retrive the corresponding `value` in $O(1)$ time complexity.
```

Comment:
The trailing "complexity" in this sentence is unnecessary, "in $O(1)$ time" is the more commonly used expression.

Reply:
From GPT-4:

In the context of computer science, the term "algorithms that run in O(1) time complexity" is more precise and authentic. It explicitly states that the time complexity of the algorithms is constant, denoted as \( O(1) \), which is a formal way to express that the execution time of the algorithms does not change with the size of the input data. This phrasing is more technically accurate and is commonly used in academic and professional discussions about algorithms.

The correct and commonly used phrase in computer science is "take \(O(1)\) time." This succinctly indicates that the algorithm or operation has a constant time complexity, meaning its execution time does not depend on the size of the input.

The second phrase, "take \(O(1)\) time complexity," while understandable, is a bit redundant because the \(O(1)\) notation inherently refers to time complexity. Thus, the preferred and more standard way to express this concept in technical and academic discussions is simply "take \(O(1)\) time."